### ISSUE TITLE
please support linux APPimage file in host system, 

### ISSUE THREAD

**Issue by newyorkthink** [winapps-org/winapps#62]

**Comment by oskardotglobal** [comment]
What do you mean by this? Would you like winapps to be packaged as an appimage?

**Comment by newyorkthink** [comment]
> What do you mean by this? Would you like winapps to be packaged as an appimage?
Yes, I like binary file and Appimage, I mean it's Winamp to appimge

**Comment by newyorkthink** [comment]
> What do you mean by this? Would you like winapps to be packaged as an appimage?

yes, i want to appimage

**Comment by FrostKnight** [comment]
I second this
